README.md

```diff
@@ -119,28 +119,71 @@ multiple labels can refer to the same bundle via its commit hash.
 
 Current use of Datamon at One Concern with respect to intra-Argo workflow usage relies on the
 [kubernetes sidecar](https://kubernetes.io/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume/)
-pattern where a shared volume is used as the transport layer for application layer
+pattern wherein a shared volume (transport layer) ramifies application layer
 communication to coordinate between the _main container_, where a data-science program
 accesses data provided by Datamon and produces data for Datamon to upload, and the
-_sidecar container_, where Datamon provides data for access (via streaming through
-main memory directly from GCS) and then, after the main container is done outputting
-data to a shared Kubernetes volume, uploads the results of the data-science program
-to GCS.  Ensuring that, for example, the streaming data is ready for access (sidecar to
-main-container messaging) as well as notification that the data-science program has
-produced output data to upload (main-container to sidecar messaging), is the responsibility
-of a couple of shell scripts that both ship inside the `gcr.io/onec-co/datamon-fuse-sidecar`
-container, which is versioned along with
+_sidecar container_, where Datamon provides data for access (as hierarchical filesystems,
+as SQL databases, etc.).
+After the main container's DAG-node-specific data-science program outputs data
+(to shared Kubernetes volume, to a PostgreSQL instance in the sidecar, and so on),
+the sidecar container uploads the results of the data-science program to GCS.
+
+Ensuring that data is ready for access (sidecar to main-container messaging)
+as well as notification that the data-science program has
+produced output data to upload (main-container to sidecar messaging),
+is the responsibility of a few shell scripts shipped as part and parcel of the
+Docker images that practicably constitute sidecars.
+While there's precisely one application container per Argo node,
+a Kubernetes container created from an arbitrary image,
+sidecars are additional containers in the same Kubernetes pod
+-- or Argo DAG node, we can say, approximately synonymously --
+that concert datamon-based data-ferrying setups with the application container.
+
+_Aside_: as additional kinds of data sources and sinks are added,
+we may also refer to "sidecars" as "batteries," and so on as semantic drift
+of the shell scripts shears away feature creep in the application binary.
```

Comment:
_aside_ : the semantic drift is such that I got eventually lost in the middle of this sentence. Probably because I've no idea what a battery could be in this context.

Reply:
Reading ahead a bit, I understand that what you are basically saying here is that the postgres-specific wrapper written in shell will gradually evolve toward native golang. Am I correct?

Reply:
see above re https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain